File: Pandaya/app/src/main/java/com/creator/pandaya/util/ProgressUtil.java
package com.creator.pandaya.util;

import android.app.Dialog;
import android.content.Context;
import android.view.Window;

import com.creator.pandaya.R;

/**
 * Created by skcrackers on 5/27/16.
 */
public class ProgressUtil extends Dialog {
    public ProgressUtil(Context context) {
        super(context);
        requestWindowFeature(Window.FEATURE_NO_TITLE);
        setContentView(R.layout.view_progress_util);
    }
}

File: 2024/OurLiving/app/src/main/java/com/creator/ourliving/util/ProgressUtil.kt
package com.creator.ourliving.util

import android.app.Dialog
import android.content.Context
import android.view.Window
import com.creator.ourliving.R

/**
 * Created by skcrackers on 5/27/16.
 */
class ProgressUtil(context: Context?) : Dialog(context!!) {
    init {
        requestWindowFeature(Window.FEATURE_NO_TITLE)
        setContentView(R.layout.view_progress_util)
    }
}